src/main/java/util/RemindChecker.java

```diff
@@ -1,42 +0,0 @@
-package util;
-
-import events.Reminder;
-import net.dv8tion.jda.api.JDA;
-import net.dv8tion.jda.api.entities.User;
-
-import java.time.LocalDateTime;
-import java.time.format.DateTimeFormatter;
-import java.util.TimerTask;
-
-public class RemindChecker extends TimerTask {
-    private JDA api;
-
-    public RemindChecker(JDA api) {
-        this.api = api;
-    }
-    @Override
-    public void run() {
-        LocalDateTime timer = LocalDateTime.now();
-        DateTimeFormatter dateTimeFormat = DateTimeFormatter.ofPattern("HH:mm dd-MM-yyyy");
-        String actualFormatedTime = timer.format(dateTimeFormat);
-
-        for(int i = 0; i < Reminder.reminders.size(); i++) {
-            if (Reminder.reminders.get(i).get(3).equals(actualFormatedTime)) {
```

Comment:
Czy to sprawdza czy już nadszedł czas na wywołanie/wysłanie przypominajki?
Co jeżeli będzie jakiś blok i nie wskoczy w odpowiednią minutę/nie pokryje się minuta z ustawioną przypominajką?

Podpowiedź:
! LocalDateTime::isBefore oraz LocalDateTime::isAfter